src/lib/holocene/badge.svelte

```diff
@@ -41,26 +41,26 @@
 
   .error,
   .red {
-    @apply border-red-700 bg-red-50 text-red-700;
+    @apply border-red-700 bg-red-300 text-black;
   }
 
   .warning,
   .yellow {
-    @apply border-yellow-900 bg-yellow-100 text-yellow-900;
+    @apply border-yellow-900 bg-yellow-200 text-black;
   }
 
   .default,
   .gray {
-    @apply bg-badge text-primary;
+    @apply bg-badge text-primary dark:bg-slate-100 dark:text-black;
   }
 
   .active,
   .available,
   .green {
-    @apply border-green-700 bg-green-100 text-green-700;
+    @apply border-green-700 bg-[#71D375] text-black;
```

Comment:
Does `#71D375` replace our existing green-100? I'd rather not use one off colors and have this added to the theme instead.

Reply:
Oh yes, I definitely meant to delete this before I pushed, design is still working on this one. Thank you.